scripts/Dockerfile.devel

```diff
@@ -1,10 +1,15 @@
 FROM ceph/ceph:v15
 
-ENV GOPATH=/go \
-	GO111MODULE=on
+
+RUN \
+ mkdir -p /go && \
+ curl https://storage.googleapis.com/golang/go1.14.1.linux-amd64.tar.gz | tar xvzf - -C /go --strip-components=1
```

Comment:
Is there a requirement for golang-1.14? Adding the version and architecture here makes building on standard environments more difficult again :cry:

Reply:
@nixpanic updated the PR description why I am updating it. May be 1.13.x is enough. lets see.

Reply:
1.13 should be good for kube 1.18. but still, you need to install go1.13 as in ceph:v15 docker image the default installation of go is 1.12.x